Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/GroupControllerTest.java

```diff
@@ -187,8 +196,12 @@ public void GivenTwoStudentsAddedIntoCreatedGroup_WhenDeleteGroupById_ShouldDele
 
         deleteGroupById(groupId);
 
+        TestTransaction.flagForCommit();
+        TestTransaction.end();
+        TestTransaction.start();
         Student student = studentRepository.findStudentById(studentA.id).orElseThrow(NotFoundException::new);
         assertEquals(0, student.getGroups().size());
+        TestTransaction.end();
```

Comment:
Wrap the following section into a method `anotherTransaction` in the superclass (`AbstractSprintBootTest`)
```
TestTransaction.flagForCommit();
TestTransaction.end();
TestTransaction.start();
....
TestTransaction.end();
```


so that I can rewrite it to

```java
anotherTransaction(() -> {
    Student student = studentRepository.findStudentById(studentA.id).orElseThrow(NotFoundException::new);
     assertEquals(0, student.getGroups().size());
})
```